src/js/src/components/ngrams/netarchive/SearchBox.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="searchBoxContainer">
+    <form class="searchForm" @submit.prevent="submitSearch">
+      <input id="query"
+             v-model="searchQuery"
+             type="text"
+             autofocus
+             placeholder="Enter search term">
+      <button id="querySubmit" title="Search" type="submit">
+        <div id="magnifyingGlass" />
+      </button>
+      <button v-if="searchQuery !== '' || Object.keys(results).length !== 0"
+              id="clearSubmit"
+              title="Clear search and results"
+              type="button"
+              @click.prevent="resetState()">
+        X
+      </button>
+    </form>
+  </div>
+</template> 
+
+<script>
+import { mapState, mapActions } from 'vuex'
+
+export default {
+  name: 'SearchBox',
+ 
+  data () {
+    return {    
+        searchQuery:''
+    }
+  },
+  
+  computed: {
+    ...mapState({
+     query: state => state.Ngram.query,
```

Comment:
You also don't use the query from the store here it seems. You don't have the method for updating it set in your methods picked from the Ngram store in mapActions.

Reply:
👍